astroid/brain/brain_pathlib.py

```diff
@@ -21,7 +21,8 @@
 
 def _looks_like_parents_subscript(node: nodes.Subscript) -> bool:
     if not (
-        isinstance(node.value, nodes.Attribute) and node.value.attrname == "parents"
+        (isinstance(node.value, nodes.Attribute) and node.value.attrname == "parents")
+        or isinstance(node.value, nodes.Name)
```

Comment:
I'm not sure we can merge this without a benchmark. Traditionally we check a specific name like "parents" instead of inferring every single name node to check for some edge case like this.

@Pierre-Sassoulas what do you think?

Reply:
We could check for the specific name "parents" but not anything else.

Reply:
Yes, I think we do need a benchmark in this case.